oauth2client/contrib/django_util/__init__.py

```diff
@@ -210,6 +270,23 @@ def _get_oauth2_client_id_and_secret(settings_instance):
                 "GOOGLE_OAUTH2_CLIENT_SECRET in settings.py")
 
 
+def _get_storage_model():
+    """This configures whether the credentials will be stored in the session
```

Comment:
I'm a bit confused, I would think every function with a leading underscore is intended to be private and so primarily for maintainers, but in many places in this repo docstrings are still used for private functions. For example, `oauth2.client:_update_query_params` or `oauth2.contrib.multiprocess_file_storage.py:def _create_file_if_needed(filename)` and several others have doc strings. Do you think those docstrings should get converted to block comments or am I misunderstanding something?